optimum/exporters/onnx/model_configs.py

```diff
@@ -624,3 +633,123 @@ def generate_dummy_inputs(self, framework: str = "pt"):
+        num_choices: int = 4,
+        random_batch_size_range: Optional[Tuple[int, int]] = None,
+        random_sequence_length_range: Optional[Tuple[int, int]] = None,
+        random_num_choices_range: Optional[Tuple[int, int]] = None,
+    ):
+        super().__init__(
+            task,
+            normalized_config,
+            batch_size=batch_size,
+            sequence_length=sequence_length,
+            num_choices=num_choices,
+            random_batch_size_range=random_batch_size_range,
+            random_sequence_length_range=random_sequence_length_range,
+            random_num_choices_range=random_num_choices_range,
+        )
+
+        self.hidden_size = normalized_config.hidden_size
+
+    def generate(self, input_name: str, framework: str = "pt"):
+        if input_name != "encoder_hidden_states":
+            dummy_input = super().generate(input_name, framework=framework)
+        else:
+            shape = (self.batch_size, self.sequence_length, self.hidden_size)
+            dummy_input = (self.random_float_tensor(shape, min_value=0, max_value=1, framework=framework), None, None)
+
+        return dummy_input
+
+
+class SpeechSeq2SeqEncoderOnnxConfig(AudioOnnxConfig):
+    NORMALIZED_CONFIG_CLASS = NormalizedAudioConfig
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return {
+            "input_features": {0: "batch_size", 1: "feature_size", 2: "encoder_sequence_length"},
+        }
+
+
+class SpeechSeq2SeqDecoderOnnxConfig(Seq2SeqOnnxConfig):
+    NORMALIZED_CONFIG_CLASS = NormalizedSeq2SeqConfig
+
+    DUMMY_INPUT_GENERATOR_CLASSES = (
+        Seq2SeqDecoderTextInputGenerator,
+        DummyDecoderTextInputGenerator,
+        DummySeq2SeqPastKeyValuesGenerator,
+    )
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        from collections import OrderedDict
+
+        common_inputs = {
+            "input_ids": {0: "batch_size", 1: "past_decoder_sequence_length + sequence_length"},
+            "encoder_hidden_states": {0: "batch_size", 1: "encoder_sequence_length"},
+        }
+
+        if self.use_past:
+            self.add_past_key_values(common_inputs, direction="inputs")
+
+        return common_inputs
+
+    @property
+    def outputs(self) -> Mapping[str, Mapping[int, str]]:
+        common_outputs = super(OnnxConfigWithPast, self).outputs
+        self.add_past_key_values(common_outputs, direction="outputs")
+        return common_outputs
+
+    def generate_dummy_inputs(self, framework: str = "pt"):
+        dummy_input = super().generate_dummy_inputs(framework)
+
+        common_inputs = {}
+        common_inputs["decoder_input_ids"] = dummy_input.pop("input_ids")
+        common_inputs["encoder_outputs"] = dummy_input.pop("encoder_hidden_states")
+
+        if "past_key_values" in dummy_input:
+            common_inputs["past_key_values"] = dummy_input.pop("past_key_values")
+
+        return common_inputs
+
+    @property
```

Comment:
Does it depend to `use_past` or not?

Reply:
Same reason as above. To always output `hidden_states` in output the following function is required. `Note: ` When `use_past` is `False` then the hidden states are present in the output but the input do not contain the `past_key_values`.